styles.css

```diff
@@ -70,3 +69,128 @@ a {
 ul {
   padding-inline-start: 0;
 }
+
+.works {
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+}
+
+.port1 {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  max-width: 327px;
+  margin: auto;
+  margin-bottom: 70px;
+}
+
+.title {
+  font-weight: 700;
+  font-size: 32px;
+  color: #172b4d;
+}
+
+.clnt {
+  font-weight: 600;
+  font-size: 13px;
+  color: #344563;
+}
+
+.deets {
+  font-family: 'Poppins', sans-serif;
+  font-weight: bold;
+  color: #7a869a;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 205px;
+  padding: 0;
+  list-style: none;
+  gap: 8px;
+}
+
+.tgs {
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  color: #6070ff;
+}
+
+.prjctbtn {
+  display: flex;
+  align-items: center;
+  background: none;
+  border: 2px solid #396df2;
+  border-radius: 10px;
+  padding: 8px 16px;
+}
+
+.prjctbtn:hover {
+  background: #396df2;
+  color: #fff;
+}
+
+.btntxt {
+  font-weight: 500;
+  font-size: 17px;
+  color: #396df2;
+  text-align: center;
+  letter-spacing: 0.03em;
+}
+
+.btntxt:hover {
+  background: #396df2;
+  color: #fff;
+}
```

Comment:
Hey I see the work you did here 💯  but this is still not right, keep going 💪🏽  it seems like the style of your text is overriding the style of your button, you don't need to style both the button and the style inside of it, it can all be done at once 1️⃣  Consider merging them, it could look like this:
```
.prjctbtn {
  display: flex;
  align-items: center;
  background: none;
  border: 2px solid #396df2;
  border-radius: 10px;
  padding: 8px 16px;
  font-weight: 500;
  font-size: 17px;
  color: #396df2;
  text-align: center;
  letter-spacing: 0.03em;
}

.prjctbtn:hover {
  background: #396df2;
  color: #fff;
}
```

That also means removing the styles for `.btntxt` and `.btntxt:hover`